<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminForm</class>
 <widget class="QWidget" name="AdminForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Администратор</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Создание пользователя</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QRadioButton" name="radioStudent">
        <property name="text">
         <string>Студент</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Логин</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="createLogin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>6</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="createPassword"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Пароль</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="radioLibrarian">
        <property name="text">
         <string>Бибилиотекарь</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QRadioButton" name="radioAdmin">
        <property name="text">
         <string>Администратор</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="createUserButton">
        <property name="text">
         <string>Создать пользователя</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>Панель Администратора</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QListWidget" name="userListWidget"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="ButtonClose">
     <property name="text">
      <string>Выйти</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
